deploy/agent/clusternet_agent_deployment.yaml

```diff
@@ -63,5 +63,6 @@ spec:
             - --cluster-reg-token=$(REG_TOKEN)
             - --cluster-reg-parent-url=$(PARENT_URL)
             - --cluster-sync-mode=Dual
+            - --enable-metric-server=false
```

Comment:
Please submit a separate PR on this change.

This flag is not available in current images.

Reply:
ok